server/api/users.js

```diff
@@ -0,0 +1,8 @@
+var express = require('express');
+var router = express.Router();
+var User = require('../models/users');
+
+// GET /api/users/ 
+router.get('/', User.getAll);
```

Comment:
The router logic here needs to be built out more, and separated from the DB/querying logic. See my comment on server/models/user.js